lib/windows/launcher/models/index.js

```diff
@@ -72,6 +72,6 @@ function getImmutableApp(app) {
     });
 }
 
-export default {
-    getImmutableApp,
+export {
+    getImmutableApp as default,
```

Comment:
So if I write
```
export {
        getImmutableApp1 as default,
        getImmutableApp2,
        getImmutableApp3,
}
```
it will use getImmutableApp1 as the default when import this module, right?

Reply:
I suppose you should.